shared/rakefiles/entrypoint-common.rake

```diff
@@ -0,0 +1,11 @@
+desc "[ONLY ADMIN] Initialize GCP provider where all the projects will live"
```

Comment:
@amatas We use underline `_` as "space substitution" for other rake files: `ci_save_all.rake`, `sh_filter.rb`. I would suggest to use the same style.